src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/cart_rule.yml

```diff
@@ -1,3 +1,12 @@
+#@todo: add legacy links after page is fully migrated. (including condition groups)
```

Comment:
condition groups is not related here I think?

Reply:
btw I dont remember if we use plural cart_rules.yml or cart_rule.yml for routes filename?
Is there a convention for this @matks?

Reply:
Yeah, "condition groups" shouldn't be there.

I am not sure about file name, other routing files seem to alternate between plural and singular

Reply:
I'm afraid we have no conventions yet so there seem to be a mix indeed.

Nice catch however ! Let's decide one convention and follow it now. I personnally have no preferences. Wdyt guys ? plural or not (no bad answer, we just need to choose)

Reply:
I see that services yaml files nearly all are singular. Maybe we should do the same here?

Reply:
sounds good to me

Reply:
singular it is !